README.md

```diff
@@ -373,9 +373,12 @@ Installation may seem lengthy, but you can actually do it quickly:
 1. If you haven't already, root your device; the rooting procedure depends on your device's brand and model.
 	* You can find a guide [here](http://www.androidcentral.com/root) for most devices
 1. Enable *System settings* > *Security* > *Unknown sources*
-1. Install the [Xposed framework](http://forum.xda-developers.com/xposed/xposed-installer-versions-changelog-t2714053)
-	* Be sure to install [the latest version](http://dl.xposed.info/latest.apk)
+1. Install the [Xposed framework](http://forum.xda-developers.com/xposed)
 	* The Xposed fix is not needed anymore
+	* For Android 4.0.3 through 4.4.4 see  [this](http://forum.xda-developers.com/xposed/xposed-installer-versions-changelog-t2714053) XDA thread 
+	* For Android 5.0.x see [this](http://forum.xda-developers.com/showthread.php?t=3034811) XDA thread
+	* For Android 5.1.x see [this](http://forum.xda-developers.com/xposed/super-alpha-posted-permission-xposed-t3072979) XDA thread
+	* For Android 5.0.x Touchwiz Roms see [this](https://www.google.com/url?q=http://forum.xda-developers.com/xposed/unofficial-xposed-samsung-lollipop-t3113463&sa=U&ved=0CAUQFjAAahUKEwjw0LTOt5THAhXE2SwKHUZxB5E&client=internal-uds-cse&usg=AFQjCNGZAN6S4q81gBwcNqTfeAx9ESS4Vg) XDA thread
```

Comment:
This link looks strange, is this okay?
Else this pull request is much appreciated!